pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSinks.java

```diff
@@ -406,6 +408,12 @@ void processArguments() throws Exception {
                 sinkConfig.setConfigs(parseConfigs(sinkConfigString));
             }
 
+            if (null != secretsConfig) {
```

Comment:
to start maybe secrets can only be defined in the function config or user config. secrets might need more than just a single key=value